src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -144,6 +155,81 @@ const ChatItemMyTextV2 = ({
+                              <View style={styles.attachmentFileContent}>
+                                <Text style={styles.attachmentFileName}>{item.file_name}</Text>
+                                <View>
+                                  <Text style={styles.attachmentFileInfo}>
+                                    {formatBytes(item.file_size)} •{' '}
+                                    {item.file_name
+                                      ?.split('.')
+                                      [item.file_name?.split('.')?.length - 1]?.toUpperCase()}
+                                  </Text>
+                                </View>
+                              </View>
+                              <View style={styles.attachmentFileIcon}>
+                                <IconFile />
+                              </View>
+                            </View>
+                          </TouchableOpacity>
+                        ) : (
+                          <TouchableOpacity
+                            key={index}
+                            style={{
+                              width: `${
+                                (attachments.length >= 3 && index > 0) || attachments.length >= 4
+                                  ? 50
+                                  : 100
+                              }%`,
+                              height: `${attachments.length >= 3 ? 50 : 100 / attachments.length}%`,
+                              position: 'relative',
+                              overflow: 'hidden'
+                            }}
+                            activeOpacity={1}
+                            onPress={
+                              attachments.length > 0 && item.type !== 'gif'
+                                ? () => onOpenMediaPreview(attachments, index, navigation)
+                                : null
+                            }>
+                            {item.type !== 'file' && (
+                              <FastImage
+                                style={styles.image}
+                                source={{
+                                  uri: item.thumb_url
+                                }}
+                              />
+                            )}
+                            {attachments.length > 4 && index === 3 && (
+                              <View style={styles.moreOverlay}>
+                                <Text style={styles.moreText}>+{attachments.length - 4}</Text>
+                              </View>
+                            )}
+                            {/* Video Play Icon */}
+                            {item.video_path && (
+                              <View style={styles.moreOverlay}>
+                                <IconVideoPlay />
+                              </View>
+                            )}
+                          </TouchableOpacity>
+                        )
+                      )}
+                  </View>
+                )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This is a significant addition to handle attachments. It seems like you're handling different types of attachments (files, images, videos) and providing different interactions for each type. However, it's important to ensure that all edge cases are handled correctly. For instance, what happens if the file URL is not valid or the file cannot be opened? What if the image URL is broken? Also, consider adding error handling for these scenarios.



<!-- This is an auto-generated comment by OSS CodeRabbit -->